core/templates/filters/format-timer.pipe.ts

```diff
@@ -16,8 +16,11 @@
  * @fileoverview FormatTime filter for Oppia.
  */
 
-import { Pipe, PipeTransform } from '@angular/core';
+import { Injectable, Pipe, PipeTransform} from '@angular/core';
```

Comment:
Do we need space before `}`?

Reply:
Yes, we do need that space, I will add that.